awesomerc.lua

```diff
@@ -210,6 +210,7 @@ screen.connect_signal("request::desktop_decoration", function(s)
         }
     }
 end)
+
```

Comment:
Well spotted! This was a mistake. Should be after line 203 because 204 belongs to starting comment on line 102 ("-- {{{ Wibar") which has an empty line after, so before closing comment should also be an empty line. Empty line 510 was removed because it belongs to openin comment on line 215 ("-- {{{ Mouse bindings") and this has no empty line after. Sorry for the error on line 213!

Reply:
please fix it then :)

Reply:
Done.

Reply:
could you make both changes in the same PR please instead of opening another one? https://github.com/awesomeWM/awesome/pull/3594/files

Reply:
I tried, but didn't work. I'll try it again on desktop, maybe only a problem on my smartphone.